readme.md

```diff
@@ -1,20 +1,26 @@
 # Product cards
-Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+
+Replace `F-u-n-H-o-u-s-e` with your Github username and copy the links to Pull Request description:
+
+- [DEMO LINK](https://F-u-n-H-o-u-s-e.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://F-u-n-H-o-u-s-e.github.io/layout_product-cards/report/html_report/)
```

Comment:
```suggestion
Replace `<your_account>` with your Github username and copy the links to Pull Request description:

- [DEMO LINK](https://F-u-n-H-o-u-s-e.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://F-u-n-H-o-u-s-e.github.io/layout_product-cards/report/html_report/)
```

Reply:
You don't need to change name in description:) Also don't make other changes in this file except
demo-link and test report link

Reply:
![image](https://user-images.githubusercontent.com/78905554/219026484-76094793-366b-495d-85e9-548bd3b7896d.png)

Reply:
- [DEMO LINK](https://F-u-n-H-o-u-s-e.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://F-u-n-H-o-u-s-e.github.io/layout_product-cards/report/html_report/)